src/App.tsx

```diff
@@ -15,36 +15,91 @@
   'Garlic',
 ];
 
+type SelectedFilter = 'alphabetically' | 'length' | 'reverse' | '';
```

Comment:
Consider using an `enum` for the `SelectedFilter` type instead of string literals to improve code readability and maintainability. This aligns with the task requirements.